logstash-core/lib/logstash/runner.rb

```diff
@@ -330,6 +330,8 @@ def execute
     end
 
     logger.info("Starting Logstash", "logstash.version" => LOGSTASH_VERSION, "jruby.version" => RUBY_DESCRIPTION)
+    jvmArgs = ManagementFactory.getRuntimeMXBean().getInputArguments()
+    logger.info "JVM bootstrap flags: #{jvmArgs}"
```

Comment:
Is `INFO` an appropriate level for this? Do we ever have credentials or other sensitive information provided via system properties?

Reply:
This PR born from from the [comment](https://github.com/elastic/logstash/pull/13525#discussion_r770581435), where the content of `jvm.options` was dumped on console. So respect to that this PR maintain almost the same behavior.
It could be that `jvm.option` contains sensitive information, but in this case it should be responsibility of the user to use the SecretStore.
About the info logging level was chosen to have the same behavior, in the other PR the content was dumped to the console.
Elasticsearch dumps some JVM options when start, at `INFO` level, so we could be consistent:
```
[2022-01-04T08:54:14,947][INFO ][o.e.n.Node               ] [kalispera] JVM arguments [-Xshare:auto, -Des.networkaddress.cache.ttl=60, -Des.networkaddress.cache.negative.ttl=10, -XX:+AlwaysPreTouch, -Xss1m, -Djava.awt.headless=true, -Dfile.encoding=UTF-8, -Djna.nosys=true, -XX:-OmitStackTraceInFastThrow, -Dio.netty.noUnsafe=true, -Dio.netty.noKeySetOptimization=true, -Dio.netty.recycler.maxCapacityPerThread=0, -Dio.netty.allocator.numDirectArenas=0, -Dlog4j.shutdownHookEnabled=false, -Dlog4j2.disable.jmx=true, -Djava.locale.providers=SPI,COMPAT, --add-opens=java.base/java.io=ALL-UNNAMED, -XX:+UseConcMarkSweepGC, -XX:CMSInitiatingOccupancyFraction=75, -XX:+UseCMSInitiatingOccupancyOnly, -Djava.io.tmpdir=/tmp/elasticsearch-6696205400317686890, -XX:+HeapDumpOnOutOfMemoryError, -XX:HeapDumpPath=data, -XX:ErrorFile=logs/hs_err_pid%p.log, -Xlog:gc*,gc+age=trace,safepoint:file=logs/gc.log:utctime,pid,tags:filecount=32,filesize=64m, -Xms1024m, -Xmx1024m, -XX:MaxDirectMemorySize=536870912, -Des.path.home=/home/andrea/workspace/elastic_products/elasticsearch-7.14.0, -Des.path.conf=/home/andrea/workspace/elastic_products/elasticsearch-7.14.0/config, -Des.distribution.flavor=default, -Des.distribution.type=tar, -Des.bundled_jdk=true]
```

Reply:
> Do we ever have credentials or other sensitive information provided via system properties?

I don't think core or any plugin leverages system properties for sensitive data, and they certainly shouldn't. There may be property files that may contain passwords, but they won't be included in the system properties dump